bot-lib/src/commands/aur_search.rs

```diff
@@ -0,0 +1,85 @@
+use crate::data::PoiseContext;
+use color_eyre::eyre::Result;
+use itertools::Itertools;
+use poise::{serenity_prelude as serenity, CreateReply};
+use raur::Raur;
+use std::cmp::Reverse;
+
+const BASE_AUR_URL: &str = "https://aur.archlinux.org/packages/";
+/// A simple command to search the aur, you cannot display more than 20 results at a time.
+#[poise::command(slash_command, rename = "aur")]
+pub async fn aur_search(
+    ctx: PoiseContext<'_>,
+    search: String,
+    #[min = 1]
+    #[max = 20]
+    amount: usize,
+) -> Result<()> {
+    ctx.defer().await?;
+
+    let search: String = search
+        .to_uppercase()
```

Comment:
why uppercase?

Reply:
leftover from copying things, you had to_uppercase() on the string filtering for the course_catalog code thought it might've been important to keep. If it doesn't do anything I can remove it

Reply:
remove it! it's just because course codes start with capital letters